src/index.html

```diff
@@ -3,12 +3,57 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header" >
+      <div class="container">
+        <a href="#home" class="logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="ul">
+            <li class="nav-item">
+              <a href="#" class="nav-link underline is-active">Apple</a>
+            </li>
```

Comment:
remove underline class as it's redundant. You have is-active class to make the underline
```suggestion
            <li class="nav-item">
              <a href="#" class="nav-link is-active">Apple</a>
            </li>
```